Gemfile

```diff
@@ -17,7 +17,7 @@ gem 'puma', '~> 4.1'
 gem 'bcrypt', '~> 3.1.7'
 # Use Active Storage variant
 # gem 'image_processing', '~> 1.2'
-
+gem 'simple_command'
```

Comment:
Do you actually use this gem? I've looked at its documentation, it does not seem to add much helpful functionality to me.
If you do not use it, please delete it from the Gemfile